<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InventoryForm</class>
 <widget class="QWidget" name="InventoryForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>835</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidgetInventory">
     <property name="styleSheet">
      <string notr="true">QTabWidget::tab-bar {
     alignment: center;
 }</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_8">
      <attribute name="icon">
       <iconset resource="../../../../resources/resources.qrc">
        <normaloff>:/icons/sword_16px.ico</normaloff>:/icons/sword_16px.ico</iconset>
      </attribute>
      <attribute name="title">
       <string>Equipments</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_14">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0" rowspan="2">
        <widget class="QTableWidget" name="tableWidgetEquipment">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="iconSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Item</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Quantity</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>GID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Position</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Equip</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Throw</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Delete</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_9">
      <attribute name="icon">
       <iconset resource="../../../../resources/resources.qrc">
        <normaloff>:/icons/flask_16px.ico</normaloff>:/icons/flask_16px.ico</iconset>
      </attribute>
      <attribute name="title">
       <string>Usable objects</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0" rowspan="2">
        <widget class="QTableWidget" name="tableWidgetUsableItems">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="iconSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Item</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Quantity</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>GID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Use</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Throw</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Delete</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_10">
      <attribute name="icon">
       <iconset resource="../../../../resources/resources.qrc">
        <normaloff>:/icons/apple_16px.ico</normaloff>:/icons/apple_16px.ico</iconset>
      </attribute>
      <attribute name="title">
       <string>Resources</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0" rowspan="2">
        <widget class="QTableWidget" name="tableWidgetResources">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="iconSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Item</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Quantity</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>GID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Throw</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Delete</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_11">
      <attribute name="icon">
       <iconset resource="../../../../resources/resources.qrc">
        <normaloff>:/icons/key_16px.ico</normaloff>:/icons/key_16px.ico</iconset>
      </attribute>
      <attribute name="title">
       <string>Quest items</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0" rowspan="2">
        <widget class="QTableWidget" name="tableWidgetQuestItems">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="iconSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Item</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Quantity</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>GID</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
